### PR TITLE
[Homework #5] - css-advanced by Max Hirning

### PR DESCRIPTION
**Pull request by Max-Hirning**
Added button to top and theme icon changing

### CODE REVIEW

File: homeworks/max-hirning/04-css/index.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+                        <a class="social_media_link" target="_blank" href="https://twitter.com/">
+                            <img class="social_media_link_icon" src="./assets/icons/twitter.svg" alt="twitter">
+                        </a>
+                        <a class="social_media_link" target="_blank" href="https://www.linkedin.com/in/max-hirning-2bbb03230/">
+                            <img class="social_media_link_icon" src="./assets/icons/linkedin.svg" alt="linkedin">
+                        </a>
+                        <a class="social_media_link" target="_blank" href="https://www.youtube.com/">
+                            <img class="social_media_link_icon" src="./assets/icons/youtube.svg" alt="youtube">
+                        </a>
+                    </nav>
```
Comment (ilya-ischenko) [inline homeworks/max-hirning/04-css/index.html:47]: BEM:
social-media
  social-media__link
    social-media__link-icon

**Review by ilya-ischenko — COMMENTED** [review]
1. I see that you are trying to push hw4 instead of hw5
2. In HW5 class names should follow BEM approach - https://en.bem.info/methodology/quick-start/#element. You do not follow this approach in your homework.

**Comment by Max-Hirning** [comment]
Fixed issue with BEM part.

About that, I am pushing hw4 and hw5. The thing is that I fixed hw4 by your remarks and was sure that there will anything left to correct, so I started hw5. Finished it before hw4 was pushed to main.

Hw4 is already in main branch and this pr pushes hw4(which wasn't changed) and hw5. In result nothing will break

**Comment by ilya-ischenko** [comment]
> Fixed issue with BEM part.
> 
> About that, I am pushing hw4 and hw5. The thing is that I fixed hw4 by your remarks and was sure that there will anything left to correct, so I started hw5. Finished it before hw4 was pushed to main.
> 
> Hw4 is already in main branch and this pr pushes hw4(which wasn't changed) and hw5. In result nothing will break

**Pull request closed without merging**